appuio/stardog/templates/pdb.yaml

```diff
@@ -7,3 +8,4 @@ spec:
   selector:
     matchLabels:
       app.kubernetes.io/name: {{ include "stardog.name" . }}
```

Comment:
All stardog instances have the same label name `app.kubernetes.io/name: stardog` which means this PDB will be created for each environment and will be applied to all instances. You might want to add `app.kubernetes.io/instance` to the matchLabels selector.

Reply:
:scream: 

Thanks, fixed now, I hope.